### PR TITLE
Shared resources

### LINKED ISSUE #36: make shared resources optional
- [ ]   add  e.g.``resources/scripts/vendor`` to basic app template without shared
- [ ]   add min version to shared if used
- [ ]   add extra gulp file?

### LINKED ISSUE #697: ditch jquery all together an longjump to bootstrap@5
the `exist-2.2.css` is practically defunct with the new hotness (surprise surprise). 
Instead of spending time in restoring the old exide templates into working order, long-jump to bootstrap@5 and review / refactor the inherited stylesheet.

### PR DESCRIPTION
**Pull request by duncdrum**
This removes `shared-resources` from yeoman generated app templates. it also switches to the new templating library OOTB. 

TODO: 35 tests are currently skipped, these need to be reactivated and not failing before this is in good shape. The current error is 

```
Error: Unexpected close tag
Line: 4
Column: 18
Char: >
```

across a number of files, the cause is still unclear to me, none of the changes should have anything to d with making xml such as .xconf invalid, i haven't even touched `collection.xconf`

see https://github.com/eXist-db/exist/issues/3914

close #36
close #697 
See #11

### CODE REVIEW

File: generators/app/templates/build.xml [hunk 1]
```diff
@@ -1,28 +1,34 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project default="all" name="<%- title %>">
-    <description><%- desc %></description>
-    <xmlproperty file="expath-pkg.xml"/>
-    <property name="project.version" value="${package(version)}"/>
-    <property name="project.app" value="<%- title %>"/>
-    <property name="build.dir" value="build"/>
+  <description>
+    <null>
```
Comment (joewiz) [inline generators/app/templates/build.xml:4]: @duncdrum I believe this unclosed element is the source of the error.

Comment (duncdrum) [reply generators/app/templates/build.xml:4]: Yup that’s it, no clue how that got in there.

**Comment by duncdrum** [comment]
progress #697 is still holding this back.

**Comment by duncdrum** [comment]
the images are a full-download from `shared-resources` I don't think it is worth sorting out which ones are no longer called by the css, since that will go the way of the dodo soon see #698 .

same reason i kept the svg unaltered, once the style sheet gets a revamp, and the css linting code goes into the generated apps i ll take another look. 

this just ant, the gulp changes are still WIP, and unrelated.

**Review by line-o — APPROVED** [review]
Good work.

**Pull request merged**